02-starting-with-data.Rmd

```diff
@@ -136,8 +137,27 @@ To open the dataset in RStudio's Data Viewer, use the `view()` function:
 view(surveys)
 ```
 
-> ### Note
->
+## Missing values in data
+Missing values are those not stored for a variable in the observation of interest. 
+This is a common problem across all domains of research. The reasons could be many. 
+Missing values may have a significant effect on the conclusions of the research. 
+When viewing above data, we observe a lot of cells showing ‘NA’ values. 
+Those are missing values. While care should be taken to collect data so that there 
+are no missing values, most of the times values go missing. 
+
+One of the ways to handle this is to omit those cases with the missing values. 
```

Comment:
It think it's a bit unusual to do listwise deletion at this stage of the data analysis process and for *all* variables. Normally this would be done once we know which variables are used in the analysis and only if the missingness is a problem.